register.html.slim

```diff
@@ -8,7 +8,7 @@ head
       clientId: 'web'
     });
 
-    var redirectUri = 'http://developers.redhat.com/confirmation';
+    var redirectUri = app.ssoConfig.confirmation;
```

Comment:
pull this into yours https://github.com/redhat-developer/developers.redhat.com/pull/639